packages/parcel-resolver/src/index.ts

```diff
@@ -34,7 +32,8 @@ export default new Resolver({
       }
     } catch (e) {
```

Comment:
Why aren't we logging this error? Curious about the context.

Reply:
Parcel resolver if returns null will move on to the next resolver - this is to say that if there is an error, we simply ignore it and let the rest of parcel's resolver handle the issue.